di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/service/ContraIndicatorsService.java

```diff
@@ -128,15 +124,19 @@ private ContraIndicatorEvidenceDto parseContraIndicatorEvidence(SignedJWT signed
     private List<CimitStubItem> mapToContraIndications(
             String userId,
             ContraIndicatorEvidenceDto contraIndicatorEvidenceDto,
-            Instant issuanceDate) {
+            SignedJWT signedJWT)
+            throws ParseException {
 
+        Instant issuanceDate = getIssuanceDate(signedJWT.getJWTClaimsSet().getNotBeforeTime());
+        String iss = signedJWT.getJWTClaimsSet().getClaim(ISSUER).toString();
```

Comment:
Can we use `getIssuer()` on the claims set?

Reply:
yeah, thanks